wandb/sdk/interface/artifacts.py

```diff
@@ -91,7 +96,12 @@ def from_manifest_json(cls, artifact, manifest_json) -> "ArtifactManifest":
     def version(cls):
         pass
 
-    def __init__(self, artifact, storage_policy, entries=None):
+    def __init__(
+        self,
+        artifact,
```

Comment:
Yeah, it wasn't obvious what their types were, and I didn't try very hard because I was just aiming to type the `filesync/` directory; this file was out of scope. I added this `storage_policy` type annotation just because it was related to... some annotation or other inside `filesync/`.